src/index.html

```diff
@@ -17,6 +31,83 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="index.html">
+        <img
+          src="images/logo.png"
+          alt="logotype"
+          class="logo"
+        />
+      </a>
+      <nav class="nav_list">
+        <ul>
+          <li>
+            <a
+              href="#"
+              class="nav_link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="nav_link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
```

Comment:
Using `href="#"` is generally not recommended as it can cause the page to jump to the top when clicked. Consider using `javascript:void(0);` or a valid URL if this is a placeholder.